src/main/java/io/xstefank/wildlfy/bot/config/Format.java

```diff
@@ -7,7 +7,8 @@ public final class Format {
     @JsonProperty("title-check")
     public RegexDefinition titleCheck;
 
-    public RegexDefinition description;
+    @JsonProperty("description")
```

Comment:
why do you need `@JsonProperty` here? Name is automatically deduced from the field name.

Reply:
Sorry, I forgot to remove it.